addons/website_event_sale/controllers/payment.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
```

Comment:
As there's some CI style errors
For our team at least we don't add license to new files either